ansible/roles/setup_course/files/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3-alpine
+
+WORKDIR /usr/src/app
+
+RUN apk update \
+ && apk upgrade \
+ && apk add --no-cache \
+    curl    
+
+RUN apk update \
+    && apk add --no-cache \
+    gcc \
+    libressl-dev \
```

Comment:
Organize the packages alphabetically if possible.

Reply:
Changed